config/karma/shared.karma.conf.js

```diff
@@ -77,15 +70,91 @@ function getConfig(config) {
     webpack: testWebpackConfig.getWebpackConfig(skyPagesConfig, argv),
     coverageReporter: {
       dir: path.join(process.cwd(), 'coverage'),
-      check: getCoverageThreshold(skyPagesConfig),
       reporters: [
         { type: 'json' },
         { type: 'html' },
         { type: 'text-summary' },
-        { type: 'lcov' }
+        { type: 'lcov' },
+        { type: 'in-memory' }
       ],
       _onWriteReport: function (collector) {
-        return remapIstanbul.remap(collector.getFinalCoverage());
+        onWriteReportIndex++;
```

Comment:
For consistency, mind either using an arrow function here, or switch the `_onExit` function to this style?